doc/authentication-methods/client-certificate.md

```diff
@@ -28,6 +28,13 @@ A `SASL EXTERNAL` authentication method is disabled by default.
 In order to enable it, please add [`sasl_mechanisms` option](../Advanced-configuration.md#authentication) to MongooseIM config file.
 Its value must include a `cyrsasl_external` item.
 Obviously the list may be longer, if the system should support both certificate and password based authentication.
+`SASL EXTERNAL` authentication method requires the use of digital
```

Comment:
The `SASL EXTERNAL` authentication method requires the use of digital certificates. 
alt:
The `SASL EXTERNAL` authentication method requires using the digital certificates.